### DISCUSSION TITLE
[请教]关于消息格式的问题

### DISCUSSION THREAD

**Discussion by super1207** [howmanybots/onebot#75] [想法 / 建议]
是否有办法获知当前的消息格式是数组还是字符串？（通过API）
如果对于不同的连接，能够动态指定消息格式为数组中或字符串中的一种就更好了。(建议)

**Comment by stdrc** [comment]
> 是否有办法获知当前的消息格式是数组还是字符串？（通过API）

一般的语言的 JSON 库应该可以判断 `message` 字段是数组还是字符串吧，看起来不需要每次都给出

> 如果对于不同的连接，能够动态指定消息格式为数组中或字符串中的一种就更好了。(建议)

看起来不错，等待 OneBot 实现作者们来给点建议（
